app_dart/lib/src/request_handlers/presubmit_luci_subscription.dart

```diff
@@ -66,7 +66,14 @@ class PresubmitLuciSubscription extends SubscriptionHandler {
       return Body.empty;
     }
     log.fine('Setting status: ${buildPushMessage.toJson()} for $builderName');
-    final Map<String, dynamic>? userData = jsonDecode(buildPushMessage.userData!) as Map<String, dynamic>?;
+
+    Map<String, dynamic> userData;
```

Comment:
```suggestion
    Map<String, dynamic>? userData;
```

Reply:
This is to pass the failure: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8808171824391452849/+/u/app_dart/stdout